pkgs/applications/networking/browsers/firefox/wrapper.nix

```diff
@@ -82,6 +83,7 @@ let
             ++ lib.optional sndioSupport sndio
             ++ lib.optional jackSupport libjack2
             ++ lib.optional smartcardSupport opensc
+            ++ lib.optional (cfg.speechSynthesisSupport or false) speechd
```

Comment:
There should be a default value in crg that the fallback here is not required.

Reply:
When you look at the other options above, the default values are also set with fallbacks like this. I might have not understood what you meant.

Reply:
@SuperSandro2000 Would you mind clarifying? I'm sorry, I did not understand the problem, I think.

Reply:
I think @SuperSandro2000 might have been suggesting that the default could be moved to the definition of `cfg` 
https://github.com/NixOS/nixpkgs/blob/38072997c52862b4fb6aa42ccf1cf9001fa0edbc/pkgs/applications/networking/browsers/firefox/wrapper.nix#L38
but I agree @sweenu that currently none of the existing defaults are in it:
https://github.com/NixOS/nixpkgs/blob/38072997c52862b4fb6aa42ccf1cf9001fa0edbc/pkgs/applications/networking/browsers/firefox/wrapper.nix#L65-L71

Also OOC since I'm still new to nix, if the defaults were to be moved into the definition would we need to do a merge between `config.${applicationName}` and the defaults?

Reply:
I don't think we need to be overly pedantic here. Like, what is even the benefit?